### ISSUE TITLE
Add version to the script output

### ISSUE THREAD

**Issue by tpoisot** [GEO-BON/boninabox-pipelines#115]
(session feedback)

_maybe_ we can bundle the git commit sha with the outputs so that it can be re-pulled and reproduced from the same environment?

**Comment by tpoisot** [comment]
In parallel to this, maybe we could use releases + link to zenodo to get a citeable DOI for the pipeline

**Comment by jmlord** [comment]
> In parallel to this, maybe we could use releases + link to zenodo to get a citeable DOI for the pipeline

For that part, a separate issue was created: https://github.com/GEO-BON/bon-in-a-box-pipeline-engine/issues/21

For the output: when exporting the results, we should have a kind of manifest in which we include everything we can: sha1 of commit, version of R, Julia, Python, tags of the docker runners, etc..

However, this manifest should also be visible on the output page. Maybe a section at the bottom that we could unfold?

- [ ] Add versions in run page
- [ ] Add versions in exported outputs
